### DISCUSSION TITLE
Metrics for Custom NER Model

### DISCUSSION THREAD

**Discussion by morankyle** [explosion/spaCy#8024] [Help: Coding & Implementations] [answered]
Hi.

I am attempting to test my model using the Example and Scorer, however, I am having trouble getting the list of Example objects. The model takes in one large blob of text and returns the entities within that text. 

Can anyone provide an example on how to implement?I searched a lot on internet but I didn't find anything that has helped

Thank you!!!

**Comment by polm** [comment, marked as answer]
Can you give more details about what you're trying to do? With your current description there's not much we can say. 

It would be best if you gave:

- your current code, self contained
- your expected output

**Reply by morankyle** [reply]
Yes sorry:

for eg in test_data:
         doc = nlp(eg[0])
         gold_dict = {'entities': [(ent[0], ent[1], ent[2]) for ent in eg[1]]}
         example = Example.from_dict(doc, gold_dict)
         examples.append(example)

scorer = Scorer(nlp)
scores = scorer.scores(examples)


The error I'm getting is:

'str' object cannot be interpreted as an integer 

Apologies, am unable to post the data or images of the code

**Reply by morankyle** [reply]
Was able to solve it! Thanks!
